src/main/java/hudson/plugins/build_timeout/impl/DeadlineTimeOutStrategy.java

```diff
@@ -87,7 +87,7 @@ public long getTimeOut(AbstractBuild<?, ?> build, BuildListener listener) throws
         }
 
         listener.getLogger().println(
-                Messages.DeadlineTimeOutStrategy_NextDeadline(TIMESTAMP_FORMAT.format(deadlineTimestamp
+                Messages.DeadlineTimeOutStrategy_NextDeadline(new SimpleDateFormat(TIMESTAMP_FORMAT_PATTERN).format(deadlineTimestamp
```

Comment:
I changed codes to instantiate `SimpleDateFormat` each time to use.
That may cause a little performance decreases, but shouldn't cause terrible performance issues as it's performed only when
* Builds starts.
* Parameter validations on configuration pages.